docs/docusaurus.config.js

```diff
@@ -4,7 +4,30 @@
 // There are various equivalent ways to declare your Docusaurus config.
 // See: https://docusaurus.io/docs/api/docusaurus-config
 
-import {themes as prismThemes} from 'prism-react-renderer';
+import { themes as prismThemes } from 'prism-react-renderer';
+import sidebarGenerator from './sidebars.js';
+
+/**
+ * Generate a typedoc / docusaurus config object for use as reference documentation generation options for each package.
+ * @param {string} pkg the sub-package to generate refdoc options for
+ */
+function packageReferenceOptions(pkg) {
+  return {
+    classPropertiesFormat: 'table',
```

Comment:
The various `*Format` properties and setting them to `table`, I think, is nicer than the default of a bulleted list - makes better use of horizontal space, and as we add more _kinds_ of information to our source code comments / JSdocs, these should show up as separate columns. E.g. if we add an `@examples` JSDoc tag to some method docs, that will show up as a new column in the generated docs.

Reply:
nice! I agree the table is the way to go 🎉